src/utils/textProcessing.ts

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { pixelWidthToScreenSize } from './windowSize';
+
+const SCREEN_SIZE_TO_MAX_WORDS = {
+  XL: 100,
+  LG: 70,
+  MD: 40,
+  SM: 30,
+};
+
+function truncateDescription(description: string, screenWidth: number): string {
```

Comment:
we need to have unit test for these kind of code. Maybe add another ticket in backlog if you plan to now finish your PR before your vacation

Reply:
Yep I wrote some units tests. Will commit them

Reply:
Done